services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -925,17 +980,17 @@ async def patch_project_node(
     db: ProjectDBAPI = app[APP_PROJECT_DBAPI]
 
     # 1. Check user permissions
-    _user_project_access_rights = await db.get_project_access_rights_for_user(
-        user_id, project_id
+    await check_user_project_permission(
+        app,
+        project_id=project_id,
+        user_id=user_id,
+        product_name=product_name,
+        permission="write",  # NOTE: MD: before only read was sufficient, double check this
```

Comment:
looks like fogotten TODO

Reply:
Its not, it seems before the write permission was not checked (I guess it should have been) as I am not sure whether it will not break something after I added it here, I wanted to point it out.